.gitignore

```diff
@@ -6,4 +6,6 @@ node_modules/
 .DS_Store
 .env.live
```

Comment:
could we do .env* to encapsulate everything that could come up

Reply:
made changes in upcoming commit